docs/en/stack/ml/anomaly-detection/ml-ad-job-types.asciidoc

```diff
@@ -0,0 +1,131 @@
+are analyzed in the context of all the splits to find unusual values in the 
+population. In other words, the population analysis is a comparison of an 
+individual entity against a collective model of all members in the population as 
+witnessed over time.
+
+For example, if you want to detect IP addresses with unusual request rates 
+compared to the number of requests coming from other IP addresses, you can use a 
+population job. That job has a `count` function to detect unusual number of 
+requests and the analysis is split by the `client_ip` field. In this context, an 
+event is anomalous if the request rate of an IP address is unusually high or low 
+compared to the request rate of all IP addresses in the population. The 
+population job builds a model of the typical number of requests for the IP 
+addresses collectively and compares the behavior of each IP address against that 
+collective model to detect outliers. 
+
+
+[discrete]
+[[advanced-jobs]]
+## Advanced jobs
+
+Advanced jobs give you all the flexibility that's possible in the 
+{ref}/ml-put-job.html[create {anomaly-jobs} API]. At the extreme, you can switch 
+to directly edit the JSON that will be sent to this endpoint. All the other 
+types of jobs described in this page _can_ be created as advanced jobs, but the 
+more specialized wizards make it easier to create jobs for common situations. 
+You can create an advanced job if you are familiar with all the functionality 
+that {ml} {anomaly-detect} provides and want to do something that the more 
+specialized wizards do not allow you to do.
+
+
+[discrete]
+[[categorization-jobs]]
+## Categorization jobs
+
+Categorization jobs cluster similar text values together, classify them into 
+categories and detect anomalies within the categories. Categorization works best 
+on machine-written text like log messages that typically contains repeated 
+strings of text; it does not work well on human-generated text because of its 
+high variability.
+
+The model learns the normal volume and pattern of a category over time so the 
+job can detect anomalous behavior, such as an unusual number of events in a 
+category by using the `count` function or messages that rarely occur by using 
+the `rare` function.
+
+
+[discrete]
+[[rare-jobs]]
+## Rare jobs
+
+Rare {anomaly-jobs} detect rare occurrences in time series data. Rare jobs use 
+the `rare` or `freq_rare` functions and detect such events in populations as 
+well. A _rare_ job finds events in simple time series data that occur rarely 
+compared to what the model observed over time. A _rare in a population_ job 
+finds members of a population that have rare values over time compared to the 
+other members of the population. The _frequently rare in a population_ job 
+detects such rare events that frequently occur for a member of a population 
+compared to other members. As an example, a job models URI paths and client IP 
```

Comment:
I think this example is related to the last type of rare job, but since three were mentioned in this paragraph, it might be worth saying explicitly. For example:
```suggestion
compared to other members. As an example of this last type of rare job, you can create one that models URI paths and client IP 
```